python/tvm/relay/grammar/Relay.g4

```diff
@@ -132,15 +132,21 @@ attr: CNAME '=' expr ;
 // relations: 'where' relation (',' relation)* ;
 // relation: ident '(' (type_ (',' type_)*)? ')' ;
 
+typeParamSeq
+  : '[' ']'
+  | '[' ident ']'
+  | '[' ident (',' ident)+ ']'
```

Comment:
```suggestion
  | '[' ident (',' ident)* ']'
```
to merge this case and the previous one.

Reply:
tuples need something different b/c they require a trailing comma

Reply:
Good idea. Done